docs/source/create-network.rst

```diff
@@ -19,10 +17,24 @@ This is a very lightweight subclass of the :class:`networkx.Graph` class. This m
 
 All :class:`scona.BrainNetwork` **methods** have a corresponding ``scona`` **function**. So while the :class:`scona.BrainNetwork` methods can only be applied to :class:`scona.BrainNetwork` objects, you can find the equivalent function in ``scona`` which can be used on a regular :class:`networkx.Graph` object.
 
-For example, if ``G`` is a :class:`scona.BrainNetwork` object, you can threshold it by typing ``G.threshold(10)``. If ``nxG`` is a :class:`Networkx.Graph` you can use ``scona.threshold_graph(nxG, 10)`` to perform the same function.
-
 A :class:`BrainNetwork` can be initialised from a :class:`networkx.Graph` or from a correlation matrix represented as a :class:`pandas.DataFrame` or :class:`numpy.array`.
 
+Threshold
```

Comment:
Although I certainly think there is useful content here, I never meant for this section to be an explanation of how to call these methods, but rather an explanation of what we mean by thresholding.

Reply:
:sigh: I meant to link to multiple lines here, but I guess I don't know how to do that. This comment is meant for the threshold section. :disappointed: